server/src/main/java/org/elasticsearch/search/aggregations/metrics/PercentileRanksAggregationBuilder.java

```diff
@@ -30,19 +30,22 @@
 import org.elasticsearch.search.aggregations.AggregationBuilder;
 import org.elasticsearch.search.aggregations.AggregatorFactories.Builder;
 import org.elasticsearch.search.aggregations.AggregatorFactory;
+import org.elasticsearch.search.aggregations.metrics.PercentilesAggregationBuilder.PercentilesConfig;
```

Comment:
In conjunction with my comment below, I think it makes sense for the config object to live with the `PercentilesMethod` enum, rather than one of the builders.  Kind of arbitrary which builder it lands on.